src/gamestate/system_state.cpp

```diff
@@ -3484,6 +3484,34 @@ void state::load_scenario_data(parsers::error_handler& err) {
 		economy::update_rgo_employment(*this);
 		economy::update_factory_employment(*this);
 		economy::daily_update(*this);
+
+		// adjust rgo power to economy setup
+		for(int j = 1; j < 3; j++) {
+			for(int i = 1; i < 1000; i++) {
+				economy::update_rgo_employment(*this);
+				economy::update_factory_employment(*this);
+				economy::daily_update(*this);
+			}
+
+			world.for_each_commodity([&](dcon::commodity_id c) {
+				auto modifer = (1.f + 
+					(world.commodity_get_total_real_demand(c) + 1.f) 
+					/ (world.commodity_get_total_production(c) + 1.f));
+				world.commodity_set_rgo_amount(c,
+					world.commodity_get_rgo_amount(c) * modifer);
+			});
+
+			world.for_each_commodity([&](dcon::commodity_id c) {
+				world.commodity_set_current_price(c, world.commodity_get_cost(c));
+			});
+		}
+
+		//final adjustment of the economy
+		for(int i = 1; i < 10000; i++) {
```

Comment:
If this is to run a simulation to tweak parameters, maybe it would be possible to have this as a separate debug feature that would run the simulation then output any of the parameters it's tweaking to a file. That way mod makers could do a big simulation run to set up the economy then provide those parameters to users so they don't have to run a simulation every time they build a scenario (might also help with reproducibility since everyone would have the same set of parameters in case simulations end up with slightly different results if there's any randomness in the economy and seeds).